Source/OpenPF2Core/Public/Commands/PF2CharacterCommandInterface.h

```diff
@@ -100,6 +111,18 @@ class OPENPF2CORE_API IPF2CharacterCommandInterface : public IPF2LogIdentifiable
 	UFUNCTION(BlueprintCallable, Category="OpenPF2|Character Commands")
 	virtual EPF2CommandExecuteImmediatelyResult AttemptExecuteImmediately() = 0;
 
+	/**
+	 * Attempt to queue this command without trying to execute it first, if possible; do nothing, if not possible.
+	 *
+	 * This method only has an effect on the server. If it is called on a client, the result will be false.
```

Comment:
Add a note about what happens when the queue gets full and how the queue position preference gets handled.